src/sentry/utils/release_versions.py

```diff
@@ -0,0 +1,111 @@
+"""
```

Comment:
So, I don't particularly like this since we will never be able to remove it since it's used in a migration, but yet it's effectively temporary since runtime won't ever use it.

I'd suggest moving it into the migration itself, and we don't really haev to worry about the tests, since they only really matter that it worked once... we awkwardly don't need them forever into the future.